FakebookNotifications/FakebookNotifications.DataAccess/NotificationsRepo.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FakebookNotifications.DataAccess.Models;
+using FakebookNotifications.DataAccess.Models.Interfaces;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FakebookNotifications.DataAccess
+{
+    public class NotificationsRepo: INotificationsRepo
+    {
+        private readonly NotificationsContext _context;
+
+        public NotificationsRepo(IOptions<NotificationsDatabaseSettings> settings)
+        {
+            //create context from settings
+            _context = new NotificationsContext(settings);
```

Comment:
there's some "new is glue" issues here, decouple to allow unit testing. (eshoponweb opted for functional testing only on that locations.api) but really we can't start implementing the repository until we've defined the domain model, so this class just shouldn't be here yet.